vortex-array/src/flatten.rs

```diff
@@ -11,6 +12,7 @@ use crate::{Array, IntoArray};
 
 /// The set of encodings that can be converted to Arrow with zero-copy.
 pub enum Flattened {
+    Null(ConstantArray),
```

Comment:
This works for now, we may just want a FLUP issue to add a NullArray so we're 1:1 with Arrow

Reply:
issue: https://github.com/spiraldb/vortex/issues/348